frontend/components/slide.module.scss

```diff
@@ -0,0 +1,37 @@
+@-webkit-keyframes slide {
+  100% {
+    left: 0;
+  }
+}
+
+.slide {
+  position: absolute;
+  left: -300px;
+  -webkit-animation: slide 0.5s forwards;
+  animation: slide 0.5s forwards;
+}
+
+@keyframes slide {
+  100% {
+    left: 0px;
+  }
+}
+
+.close {
```

Comment:
Can we keep this file in alphabetical order. I'm not sure if we should separate styling related to slide, but we should have close related styling before slide related styling.